guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -54,9 +54,9 @@ On the main {Project} server:
 +
 For more information on custom SSL certificates signed by a Certificate Authority, see {InstallingSmartProxyDocURL}deploying-a-custom-ssl-certificate-to-{smart-proxy-context}-server_{smart-proxy-context}[Deploying a Custom SSL Certificate to {SmartProxyServer}] in _{InstallingSmartProxyDocTitle}_.
 +
-. Copy the resulting tarball to your {SmartProxy}, for this example we will use `/root/myproxy.example.com-certs.tar`
 endif::[]
 ifdef::katello[]
+. Copy the resulting tarball to your {SmartProxy}, for this example we will use `/root/myproxy.example.com-certs.tar`
```

Comment:
```suggestion
. Copy the resulting tarball to your {SmartProxy}, for example to `/root/myproxy.example.com-certs.tar`.
```

I think there is another PR where I've recommended using `{smartproxy-example-com}`.
See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L96

Reply:
Found it: https://github.com/theforeman/foreman-documentation/pull/1479#discussion_r930731670

Reply:
Yes, this was mostly a quick PR to show what I think is the more minimal solution to #1479.

I do think `{smartproxy-example-com}` is a good suggestion, but it deserves its own commit and probably its own PR.

Reply:
OK, then only the "we" stands.

Reply:
Yes, I just moved the sentence around. Didn't really modify it. Perhaps we can keep those stylistic changes for the other PR?